include/interface/cmdTlmServer.h

```diff
@@ -87,17 +96,27 @@ void cmdTlmServer_basicClose(serverInfo_t *pServerInfo);
 void cmdTlmServer_close(serverInfo_t *pServerInfo);
 
 // Send a command status to the message queue. The arguments are:
-// - server information
-// - counter value of command being acknowledged
-// - command status (enum: 'CmdStatus')
-// - estimated duration (seconds); 0 if already done
-// - reason the command failed... (put "" if nothing to report. If "reason" is
-//   longer than the buffer defined in commandStatusStructure_t, it will be
-//   truncated to fit.)
+// - pServerInfo: pointer to the server information
+// - counter: counter value of command being acknowledged
+// - cmdStatus: command status (enum: 'CmdStatus')
+// - duration: estimated duration (seconds); 0 if already done
+// - pReason: reason the command failed... (put "" if nothing to report. If
+//   "pReason" is longer than the buffer defined in commandStatusStructure_t,
+//   it will be truncated to fit.)
 // Return 0 if success, otherwise, return -1.
 int cmdTlmServer_sendCmdStatusToMsgQueue(serverInfo_t *pServerInfo,
                                          unsigned int counter,
                                          unsigned int cmdStatus,
-                                         double duration, const char *reason);
+                                         double duration, const char *pReason);
+
+// Send a telemetry message to the message queue. The arguments are:
+// - pServerInfo: pointer to the server information
+// - pMsg: pointer to the telemetry message
+// - sizeMsg: size of the message in bytes
+// - priority: A nonnegative integer that specifies the priority of this
+//   message. See the mq_send() for the details.
+// Return 0 if success, otherwise, return -1.
+int cmdTlmServer_sendTlmToMsgQueue(serverInfo_t *pServerInfo, const char *pMsg,
```

Comment:
Do you have a use case for sending messages out of order? If not, I suggest eliminating the priority argument and using a fixed constant priority. 

I can imagine possibly wanting command responses to be higher priority than telemetry, but I hope the system will be fast enough that there is no need for this, and you can simply send messages in order. It will make it easier to understand what's going on.

Reply:
I don't see any use of this function in this codebase, except in a unit test. Is it called in the higher-level packages?

Reply:
This function will be called by the `ts_rotator_controller` and `ts_hexapod_controller`.

Reply:
Removed the priority argument in commit: 7473889. Thanks!